src/integration-test/java/com/scalar/db/storage/cosmos/CosmosIntegrationTest.java

```diff
@@ -60,6 +68,17 @@ public void setUp() throws Exception {
         new CosmosContainerProperties(TABLE, PARTITION_KEY);
     client.getDatabase(KEYSPACE).createContainerIfNotExists(containerProperties);
 
+    String storedProcedure =
+        Files.lines(Paths.get(STORED_PROCEDURE_PATH), StandardCharsets.UTF_8)
+            .reduce("", (prev, cur) -> prev + cur + System.getProperty("line.separator"));
+    CosmosStoredProcedureProperties properties =
+        new CosmosStoredProcedureProperties("mutate.js", storedProcedure);
+    client
+        .getDatabase(KEYSPACE)
+        .getContainer(TABLE)
+        .getScripts()
+        .createStoredProcedure(properties, new CosmosStoredProcedureRequestOptions());
```

Comment:
I added a stored procedure registration because the simple `put()` requires the stored procedure.